src/override_font_weight.js

```diff
@@ -0,0 +1,49 @@
+await new Promise((resolve) =>
+  document.readyState === 'loading'
+    ? document.addEventListener('readystatechange', resolve, { once: true })
+    : resolve()
+);
+
+export const fontWeightOverride = Object.assign(document.createElement('style'), {
+  id: 'palettes-for-tumblr-override'
+});
+
+const getSelectorsFromUrl = async (url) => {
+  const cssText = await fetch(url).then(response => response.text());
+  const sheet = await new CSSStyleSheet().replace(cssText);
+
+  const allCssStyleRules = [...sheet.cssRules]
+    .filter((rule) => rule instanceof CSSStyleRule && rule.style)
+    .map(({ selectorText, style }) => ({
+      selectorText,
+      rules: Object.fromEntries([...style].map((key) => [key, style.getPropertyValue(key)]))
```

Comment:
You know, I guess I totally don't have to actually do this; we're only reading two specific properties. I just absolutely hate dealing with (logging, for example) the [CSSStyleDeclaration interface](https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleDeclaration), which is just, like... I mean, come on, why.